crawlers/mooncrawl/mooncrawl/stats_worker/queries.py

```diff
@@ -86,27 +83,36 @@ def data_generate(
     process_session = sessionmaker(bind=engine)
     db_session = process_session()
 
+    metadata = {
+        "source": "drone-query-generation",
+        "query_id": query_id,
+        "file_type": file_type,
+        "params_hash": params_hash,
+        "params": json.dumps(params),
+    }
+
     try:
+
+        # TODO:(Andrey) Need optimization that information is usefull but incomplete
+        block_number, block_timestamp = db_session.execute(
+            "SELECT block_number, block_timestamp FROM polygon_labels WHERE block_number=(SELECT max(block_number) FROM polygon_labels where label='moonworm-alpha') limit 1;",

```

Comment:
What if it is an Ethereum query?

Reply:
Rigth now in question TODO exactly about it)